.gitignore

```diff
@@ -3,3 +3,4 @@
 /telegraf.exe
 /telegraf.gz
 /vendor
+.vscode/launch.json
```

Comment:
Please don't add these editor specific in our project, but on your global `.gitignore` file.